CallAutomation_MediaStreaming/Readme.md

```diff
@@ -0,0 +1,87 @@
+API first answers the call with Media Streaming options settings. Once call connected, external PSTN user say something.  
+The audio is streamed to WebSocket server and generates log events to show media streaming is happening on the server.
+It supports Audio streaming only (mixed/unmixed format).
+
+This sample has 3 parts:
+
+1. ACS Resource IncomingCall Hook Settings, and ACS acquired Phone Number.
+2. IncomingCall WebApp - for accepting the incoming call with Media Options settings.
+3. WebSocketListener – Listen to media stream on websocket.
+
+The application is an app service application built on .NET6.0.
+
+## Prerequisites
+
+- Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/)
+- [Visual Studio (2022 and above)](https://visualstudio.microsoft.com/vs/)
+- [.NET6](https://dotnet.microsoft.com/en-us/download/dotnet-framework/net48) (Make sure to install version that corresponds with your visual studio instance, 32 vs 64 bit)
+- Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You'll need to record your resource **connection string** for this sample.
+- [Configuring the webhook](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/call-automation/callflows-for-customer-interactions?pivots=programming-language-csharp#subscribe-to-incomingcall-event) for **Microsoft.Communication.IncomingCall** event.
+- Enable Visual studio dev tunneling for local development. For details, see [Enable dev tunnel] (https://learn.microsoft.com/en-us/connectors/custom-connectors/port-tunneling)
+	- To enable dev tunneling, Click `Tools` -> `Options` in Visual Studio 2022
+	- In the search bar type tunnel, Click the checkbox under `Environment` -> `Preview Features` called `Enable dev tunnels for Web Application`
+	![EnableDevTunnel](./Media/EnableDevTunnel.png)
+	- Login into your account under `Dev Tunnels` -> `General`  
+	![LogInDevTunnel](./Media/AddAccountForTunnel.png)
+
+
+## Before running the sample for the first time
+
+1. Open an instance of PowerShell, Windows Terminal, Command Prompt or equivalent and navigate to the directory that you'd like to clone the sample to.
+2. git clone https://github.com/Azure-Samples/Communication-Services-dotnet-quickstarts.git.
+
+### Locally running the media streaming WebSocket app
+
+1. Go to CallAutomation_MediaStreaming folder and open `IncomingCallMediaStreaming.sln` solution in Visual Studio.
+2. Select and run the `WebSocketListener` project, an application for listening media stream on websocket.
+3. Used the dev tunnel to get public URL for `WebSocketListener` app, as this websocket URL needed for `MediaStreamingTransportURI` configuration.
+
+### Publish  the Incoming call media streaming to Azure WebApp
```

Comment:
Is it possible for Contoso to run both the WebSocket listener and call handlers locally?